drake-torch.dockerfile

```diff
@@ -171,7 +171,9 @@ RUN python3 -m pip install --upgrade --no-cache-dir --compile \
         supervisor \
         sphinx \
         sphinx_rtd_theme \
-        click
+        click \
+        pyrealsense2==2.38.1.2225 \
```

Comment:
pyrealsense doesn't have a PyPi package for python 3.8 so if our docker image ever uses this as the default python version we might consider including a step to build from source